src/containers/app/AppSagas.js

```diff
@@ -176,14 +179,53 @@ function* switchOrganizationWorker(action :Object) :Generator<*, *, *> {
   yield put(push(Routes.HOME_PATH));
 }
 
+/*
+ * initializeApplication()
+ */
+
+function* initializeApplicationWorker(action :SequenceAction) :Generator<*, *, *> {
+
+  try {
+    yield put(initializeApplication.request(action.id));
+
+    // Load app
+    yield all([
+      call(loadAppWorker, loadApp()),
+      call(getAllPropertyTypesWorker, getAllPropertyTypes())
```

Comment:
Why are you specifically loading the property types? Also, why not the full thing, i.e. `getEntityDataModel`?

Reply:
I believe the `AppApiSagas` that fire from `loadApp` gets all of the entity sets and associations necessary. just getting the property types is just to fill in the gap.